CHANGELOG.md

```diff
@@ -31,8 +31,17 @@ request adding CHANGELOG notes for breaking (!) changes and possibly other secti
 
 ### Upgrade notes
 
+- JDBC persistence: a new column has been added to the `events` table to store the OpenTelemetry 
+  context that was active when the event was emitted. This column is nullable. To update your schema
+  to the latest version, run the following SQL statement: 
+  ```sql 
+  ALTER TABLE POLARIS_SCHEMA.EVENTS ADD COLUMN IF NOT EXISTS otel_context TEXT; 
+  ```
+
 ### Breaking changes
 
+- The default request ID header name has changed from `Polaris-Request-Id` to `x-request-id`.
```

Comment:
Can we also change it in the doc page https://polaris.apache.org/in-dev/unreleased/telemetry?

Reply:
Good catch! Will do.